examples/persistent-storage/efr32/third_party/connectedhomeip

```diff
@@ -0,0 +1 @@
+/Users/GitHub/connectedhomeip
```

Comment:
Is this needed?
Path seems to be MacOS (?) specific. If needed, could we use a relative link instead?

Reply:
fixed to relative symlinks